examples/fuzzing/calculator_test.go

```diff
@@ -0,0 +1,79 @@
+package main
+
+/* Fuzz testing or fuzzing is an automated software testing method
+that injects invalid, malformed, or unexpected inputs into a system
+to reveal software defects and vulnerabilities.
+A fuzzing tool injects these inputs into the system and then monitors for exceptions such as crashes or information leakage
+*/
+
+//A simple way of using fuzzing to test a program using a claculator instance
+import (
+	"errors"
+	"fmt"
+	"testing"
+)
+
+/*A calculator for performing simple arithmetic operations*/
+func calculator(num1, num2 float64, operator string) (float64, error) {
+	var result float64
+
+	switch operator {
+	case "-":
+		result = num1 - num2
+
+	case "+":
+		result = num1 + num2
+
+	case "/":
+		if num2 == 0 || num1 == 0 {
+			return 0, errors.New("Division by 0")
+		}
+	case "*":
+		result = num1 * num2
+	}
+
+	return result, nil
+}
+
+/*Accepts inputs, calls the calculator function and executes inputs and return results*/
+func InputCalculator(data float64) float64 {
```

Comment:
Not sure what's going on here. `InputCalculator` does not appear to be using the `data` parameter at all - what's doing the fuzzing here?